test/js_interop_helpers_test/shared_tests.dart

```diff
@@ -49,7 +57,7 @@ void sharedJsFunctionTests() {
 
     group('markChildValidated', () {
       test('is function that does not throw when called', () {
-        expect(() => markChildValidated(new EmptyObject()), returnsNormally);
+        expect(() => markChildValidated(newObject()), returnsNormally);
```

Comment:
lets actually leave both tests in for now until its fully removed.
```suggestion
        expect(() => markChildValidated(new EmptyObject()), returnsNormally);
        expect(() => markChildValidated(newObject()), returnsNormally);
```